code/modules/sanity/breakdowns.dm

```diff
@@ -348,79 +348,253 @@
+	return FALSE
+
+/datum/breakdown/common/power_hungry/occur()
+	RegisterSignal(holder.owner, COMSIG_CARBON_ELECTROCTE, .proc/check_shock)
+	RegisterSignal(holder.owner, COMSIG_LIVING_STUN_EFFECT, .proc/check_shock)
+	return ..()
+
+/datum/breakdown/common/power_hungry/conclude()
+	UnregisterSignal(holder.owner, COMSIG_CARBON_ELECTROCTE)
+	UnregisterSignal(holder.owner, COMSIG_LIVING_STUN_EFFECT)
+	..()
+
+/datum/breakdown/common/power_hungry/proc/check_shock()
+	finished = TRUE
+
+
+/datum/breakdown/negative/glassification
+	name = "Glassification"
+	duration = 5 MINUTES
+	restore_sanity_post = 40
+	/*start_messages = list(
+		"You feel like there is no point in any of this!",
+		"You brain refuses to comprehend any of this!",
+		"You feel like you don't want to continue whatever you're doing!",
+		"You feel like your best days are gone forever!",
+		"You feel it. You know it. There is no turning back!"
+	)
+		end_messages = list(
+		"You feel at ease again, suddenly."
+	)*/
+	var/time
+	var/cooldown = 15 SECONDS
+	var/time_view = 5 SECONDS
+	var/active_view = FALSE
+	var/mob/living/carbon/human/target
+	start_messages = list("You start to see through everything. Your mind expands.")
+	end_messages = list("The world has returned to normal ... right?")
+
+/datum/breakdown/negative/glassification/can_occur()
+	var/list/candidates = (GLOB.player_list & GLOB.living_mob_list & GLOB.human_mob_list) - holder.owner
+	if(candidates.len)
+		return TRUE
+	return FALSE
+
+/datum/breakdown/negative/glassification/update()
+	if(world.time < time)
+		return TRUE
+	if(active_view)
+		holder.owner.remoteviewer = FALSE
+		holder.owner.remoteview_target = null
+		holder.owner.reset_view(0)
+		target.remoteviewer = FALSE
+		target.remoteview_target = null
+		target.reset_view(0)
+		target = null
+		active_view = FALSE
+		target = null
+		time = world.time + cooldown
+		return TRUE
+	. = ..()
+	if(!.)
+		return FALSE
+	var/list/targets = (GLOB.player_list & GLOB.living_mob_list & GLOB.human_mob_list) - holder.owner
+	if(targets.len)
+		target = pick(targets)
+		holder.owner.remoteviewer = TRUE
+		holder.owner.remoteview_target = target
+		holder.owner.reset_view(target)
+		to_chat(holder.owner, SPAN_WARNING("It seems as if you are looking through someone else's eyes."))
+		to_chat(target, SPAN_WARNING("It seems as if you are looking through someone else's eyes."))
+		target.remoteviewer = TRUE
+		target.remoteview_target = holder.owner
+		target.reset_view(holder.owner)
+		target.sanity.changeLevel(-rand(5,10))
+		active_view = TRUE
+		time = world.time + time_view
+
+/datum/breakdown/common/herald
+	name = "Herald"
+	restore_sanity_pre = 5
+	restore_sanity_post = 45
+	duration = 5 MINUTES
+	start_messages = list("You've seen the abyss too long, and now forbidden knowledge haunts you.")
+	end_messages = list("You feel like you've forgotten something important. But this comforts you.")
+	var/list/messages = list("The Feast is over, and the lamps expire.",
```

Comment:
No, as was specified in design documentation, there is a much larger list of sanity messages that is shown when sanity is low. You should use it here. If it's not assailable for any reason - make it global.